src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -111,4 +111,25 @@ class Wallet(
   fun removeAccountSigner(sourceAddress: String, signerAddress: String): Transaction {
     return addAccountSigner(sourceAddress, signerAddress, 0)
   }
+
+  // Submit transaction
+  fun submitTransaction(signedTransaction: Transaction, serverInstance: Server = server): Boolean {
+    try {
+      val response = serverInstance.submitTransaction(signedTransaction)
+
+      if (response.isSuccess) {
+        return true
+      }
+
+      var errorMessage = "Transaction failed"
+
+      if (!response.extras?.resultCodes?.transactionResultCode.isNullOrBlank()) {
+        errorMessage += ": ${response.extras.resultCodes.transactionResultCode}"
+      }
+
+      throw Exception(errorMessage)
+    } catch (e: Throwable) {
+      throw e
```

Comment:
We shouldn't have a try/catch at all if we just want to re-throw it.

If we decide we do want to do something about the error then we can keep the try/catch, but I think we need something more specific than `Throwable`. We should be catching specific errors raised from `submitTransaction()`.